build_debian.sh

```diff
@@ -616,6 +621,46 @@ if [[ ! -f './asic_config_checksum' ]]; then
 fi
 sudo cp ./asic_config_checksum $FILESYSTEM_ROOT/etc/sonic/asic_config_checksum
 
+## Check if not a last stage of RFS build
+fi
+
+if [[ $RFS_SPLIT_FIRST_STAGE == y ]]; then
+    echo '[INFO] Finished with RFS first stage'
+    echo '[INFO] Umount all'
+
+    ## Display all process details access /proc
+    sudo LANG=C chroot $FILESYSTEM_ROOT fuser -vm /proc
+    ## Kill the processes
+    sudo LANG=C chroot $FILESYSTEM_ROOT fuser -km /proc || true
+    ## Wait fuser fully kill the processes
+    sleep 15
+    sudo LANG=C chroot $FILESYSTEM_ROOT umount /proc
+
+    sudo rm -f $TARGET_PATH/$RFS_SQUASHFS_NAME
+    sudo mksquashfs $FILESYSTEM_ROOT $TARGET_PATH/$RFS_SQUASHFS_NAME
```

Comment:
Agreed. I _think_ a simple `tar` (without any compression) would serve the purpose here.

Reply:
Good point. I did some testing:
* default: 27 sec, 532M
* compression-level 1: 2 sec, 574M
* no compression: 1 sec, 1.4G

The "compression-level 1" seems to be the optimal tradeoff.

@saiarcot895 
Regarding `tar`, I'd stick with `mksquashfs` since it is already tested and (after compression tuning) performs very well.

I've updated the PR.